#!/usr/bin/env python
PACKAGE = "lla2enu"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("soglia_safe", double_t, 1, "Soglia Safe",    5, 0,   100)
gen.add("soglia_unsafe", double_t, 1, "Soglia Unsafe",    1, 0,   100)


exit(gen.generate(PACKAGE, "pub_custom_msg", "parametri"))
